homework-submission/Week1/App.js

```diff
@@ -0,0 +1,72 @@
+const ListItem = ({ description, deadline, className }) => {
+  return (
+    <li class={className}>
```

Comment:
When I have opened my console the first thing I saw is this red Warning which indicates what is says 👆 

![Screen Shot 2019-06-28 at 05 19 14](https://user-images.githubusercontent.com/25272822/60314860-73cb6680-9964-11e9-932e-984790993a64.png)